### PR TITLE
Add iproute2, iptables, and libnftnl.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] REQUIRED - The package is added to `packages.txt`

### CODE REVIEW

File: iproute2.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      - iptables-dev
+      - elfutils-dev
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://kernel.org/pub/linux/utils/net/iproute2/iproute2-${{package.version}}.tar.xz
```
Comment (luhring) [inline iproute2.yaml:24]: Would it be advantageous to use the git repo instead here? (I don't think I'm fully aware of the tradeoffs that would determine the answer to this. But I have a soft intuition that traceability into source history will be helpful over time)

(https://github.com/shemminger/iproute2)

**Review by luhring — DISMISSED** [review]

File: iproute2.yaml (hunk 1 above)
Comment (dlorenc) [reply iproute2.yaml:24]: I looked at this but wasn't terribly confident it was really maintained or official. I'm happy to switch if it's real.

**Comment by rawlingsj** [comment]
This PR https://github.com/chainguard-dev/melange/pull/392 seems to fix the 403 in the failing update check above

**Comment by rawlingsj** [comment]
the 403 has been fixed but the failed check I think I related to https://github.com/wolfi-dev/os/issues/1314. I think we can approve / merge this.

**Review by rawlingsj — DISMISSED** [review]

**Review by rawlingsj — DISMISSED** [review]

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**